### ISSUE TITLE
Lock screen colours are too bright

### ISSUE THREAD

**Issue by digitalethics** [vinceliuice/ChromeOS-kde#17]
When using the ChromOS-kde light theme, the lock screen colours are too bright. The original blur effect is gone.

**Comment by vinceliuice** [comment]
I'll fix this

**Comment by digitalethics** [comment]
@vinceliuice I know that ChromeOS-kde is probably not a high-priority item for you at the moment. Would you be interested in a joint hacking session to get a couple of issues fixed for this theme? You can email me and we can tackle this when we both have time.

**Comment by vinceliuice** [comment]
Is the background picture too bright? I can make it darker

**Comment by digitalethics** [comment]
Yes, it is too bright and the colours look overexposed. I think the Breeze theme settings are fine for this, so it would be good if ChromeOS-kde can just default to that.

**Comment by vinceliuice** [comment]
I'll update this
